packages/@react-stately/select/src/useSelectState.ts

```diff
@@ -15,7 +15,8 @@ import {FormValidationState, useFormValidationState} from '@react-stately/form';
 import {OverlayTriggerState, useOverlayTriggerState} from '@react-stately/overlays';
 import {SelectProps} from '@react-types/select';
 import {SingleSelectListState, useSingleSelectListState} from '@react-stately/list';
-import {useState} from 'react';
+import {useEffectEvent} from '@react-aria/utils';
```

Comment:
can't import from react-aria in a react-stately package

Reply:
even if this is just useEffectEvent?

Reply:
it should be moved to react-stately/utils then and re-exported by react-aria/utils
otherwise you'll get other weird dependencies